tensorflow/compiler/mlir/tosa/transforms/legalize_common.h

```diff
@@ -30,6 +30,7 @@ limitations under the License.
 
 namespace mlir {
 namespace tosa {
+constexpr char kOverrideZeropointAttrName[] = "override_zeropoint";
```

Comment:
I think you need to remove this attribute, but if not and you kept it, you should be using a namespaced attribute (i.e. `tosa._override_zeropoint`). But please, per other comments, I think we should eliminate the use entirely.

Reply:
Agree that I should keep this under tosa namespace. And let's discuss main concern in the top thread.